inscriptions/ISic000336.xml

```diff
@@ -185,14 +186,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la">
+                <ab>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan><g type="interpunct">·</g><expan><abbr>M</abbr><ex>anibus</ex></expan><g type="interpunct">·</g><expan><abbr>S</abbr><ex>acrum</ex></expan>
```

Comment:
Can you insert spaces between each separate 'word' or equivalent, so, that would be in this line after each `</expan>`, after each `</g>`